mobify/platform-scaffold
# Update and clean out web dependencies

This PR updates dependencies for the web scaffold. Notable changes are major version bumps for both `reselect` and `react-redux` with significant performance increases; the breaking changes do not affect our use case for either package.

 **JIRA**: N/A

## Changes
- Update dependencies
- Remove unused `es6-promise` dependency

## How to test-drive this PR
- npm install
- npm run test:all
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- See that nothing has changed.
